vignettes/how-to/use-shinymanager.Rmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "How-to: Use shinymanager"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How-to: Use shinymanager}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+To use [`shinymanager`](https://datastorm-open.github.io/shinymanager/) in Rhino
+you will need to setup a [legacy entrypoint](https://appsilon.github.io/rhino/reference/app.html#legacy-entrypoint)
+in your `rhino.yml`:
+
+```yml
+legacy_entrypoint: box_top_level
+```
```

Comment:
I'm not ok with this solution, since it is more a hack/workaround than providing true compatibility. `legacy_entrypoint` is named legacy for a reason. I want to avoid a situation when using it becomes a standard. This can be a sign that the decision to keep the outermost UI and server as Shiny modules was wrong.

Reply:
`shinymanager::secure_app()` is designed to be passed directly to `shinyApp()` and I don't see a clean way to put it inside a Shiny module. Thus there is no way around using `box_top_level` entrypoint. Indeed it is unfortunate that it's called "legacy" in this case.

Moving forward we might want to:
1. Drop `legacy` from the entrypoint setting, and/or
2. make `box_top_level` the default.

Both options would require a significant effort to ensure backward compatibility, update our docs, and come up with a good design in general. There are some possible variations here, e.g. we could have an `entrypoint` setting with `legacy_app_dir`, `legacy_source`, `box_top_level` and `box_shiny_module`.

IMO the best course of action would be to release it as is and consider changes to the entrypoint setting in the future. WDYT?

Reply:
I'm ok with releasing it as long as it is explicitly stated in the documentation, that this is a workaround, not a setting recommended for all cases.

Reply:
Done :+1: